apiary.apib

```diff
@@ -564,6 +564,75 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Mentors | Collection [/api/v1/mentors]
+
+### List All Mentors Available [GET]
+
```

Comment:
Both the mentors controller and the services controller have a `before_action :authenticate_user!`.  As such, you'll need a request section for authentication.  For example:

https://github.com/OperationCode/operationcode_backend/blob/master/apiary.apib#L67-L71